index.html

```diff
@@ -26,5 +26,139 @@ <h3 class="profession">I’m a Software Developer</h3>
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+        </div>          
+    </section>
+    <section id="about-section">
```

Comment:
- [ ] kindly increase the space between the about and work section like in the original design. this will help differentiate the two sections. nice work
![Screenshot (748)_LI](https://user-images.githubusercontent.com/65298048/212431291-52ea2c4d-ed83-47b2-acf1-45dbd54b5fce.jpg)